experimental/orchestrator/docs/Overview.md

```diff
@@ -0,0 +1,109 @@
+# Technical Overview
+
+The Orchestrator is a replacement of the [Bot Framework Dispatcher][1] used in chat bots since 2018. It makes the state of the art natural language understanding methods available to bot developers while at the same time making the process of language modeling quick, not requiring the expertise in [Deep Neural Networks (DNN), Transformers][5], or [Natural Language Processing (NLP)][6]. This work is co-authored with the industry experts in the field and include some of the top methods used in the [General Language Understanding Evaluation (GLUE)][7] leaderboard. Orchestrator will continue evolve and adopt the latest advancements in science and in the industry.

```

Comment:
Typos: "includes", "...will continue to evolve..."

Reply:
Fixed.